internal/helmdeployer/template.go

```diff
@@ -57,5 +57,5 @@ func Template(ctx context.Context, bundleID string, manifest *manifest.Manifest,
 		return nil, err
 	}
 
-	return resources.Objects, nil
+	return ReleaseToObjects(resources)
```

Comment:
nit: renaming `resources` to `release` could clarify this code as already done around other calls to `Deploy`.

Reply:
Thanks:)